src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -712,4 +632,44 @@ class JavaMigrations {
 		Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "14.ea.17-open", _))
 	}
 
+  @ChangeSet(order = "197", id = "197-add_zulu_7_0_242", author = "jorsol")
+  def migrate159(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "7.0.242-zulu", "https://cdn.azul.com/zulu/bin/zulu7.34.0.5-ca-jdk7.0.242-linux_x64.tar.gz", Linux64, Some(Zulu)),
+      Version("java", "7.0.242-zulu", "https://cdn.azul.com/zulu/bin/zulu7.34.0.5-ca-jdk7.0.242-win_x64.zip", Windows, Some(Zulu)),
+      Version("java", "7.0.242-zulu", "https://cdn.azul.com/zulu/bin/zulu7.34.0.5-ca-jdk7.0.242-macosx_x64.zip", MacOSX, Some(Zulu))
```

Comment:
We will need to [use gzip](https://github.com/sdkman/sdkman-hooks/blob/master/app/controllers/HooksController.scala#L35) on OSX for all Zulu binaries at this point.

Reply:
Zulu don't have a `tar.gz` for Java 7, but it actually have a `zip` for all versions, so I think that make sense to use the ZIP instead of the GZ, and change that hook to default_post_zip